project/Dependencies.scala

```diff
@@ -11,6 +13,10 @@ object Dependencies {
   def coursierV = "2.0.0-RC5-6"
   def coursierInterfaceV = "0.0.22"
   val currentScalaVersion = scala213
+  // we support 3 last binary versions of each scala Version, except for 2.11 because semanticdb-scalac_2.11.10 is missing.
+  val supportedScalaVersionforExplicitResultRule =
```

Comment:
I think this should not be specific to a single rule - we could test all of them (`unit/testOnly scalafix.tests.rule.RuleSuite`) against that set of versions (which will only be useful for those that use the presentation compiler, but at least we won't miss cross-testing it when a new one is introduced).